README.md

```diff
@@ -26,16 +26,14 @@ The CDK is available in the following languages:
 * Java ([Java ≥ 8](https://www.oracle.com/technetwork/java/javase/downloads/index.html) and [Maven ≥ 3.5.4](https://maven.apache.org/download.cgi))
 * .NET ([.NET Core ≥ 3.1](https://dotnet.microsoft.com/download))
 
--------
+\
+Jump To:
+* [Getting Started](#getting-started)
+* [Getting Help](#getting-help)
+* [Contributing](#contributing)
+* [More Resources](#more-resources)
 
-[Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide) |
-[CDK Workshop](https://cdkworkshop.com/) |
-[Getting Started](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html) |
-[API Reference](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-construct-library.html) |
-[Examples](https://github.com/aws-samples/aws-cdk-examples) |
-[Getting Help](#getting-help) |
-[RFCs](https://github.com/aws/aws-cdk-rfcs) |
-[Roadmap](https://github.com/aws/aws-cdk/blob/master/ROADMAP.md)
```

Comment:
I think we should keep this at the top (just change the link for `Getting Started` and `Roadmap` to TOC links, add `More Resources` and `Contributing`, and remove `Examples` and `CDK Workshop` in favor of `More Resources`.

There is a lot of benefit for active "less-new" users to have links to the API Reference, RFCs, and Dev Guide without scrolling